datasets/samplemotordatalimerick.py

```diff
@@ -0,0 +1,662 @@
+###################################################################################################
+#
+# Copyright (C) 2024 Analog Devices, Inc. All Rights Reserved.
+# This software is proprietary to Analog Devices, Inc. and its licensors.
+#
+###################################################################################################
+"""
+Classes and functions for the Limerick Sample Motor Data Dataset
+"""
+import errno
+import math
+import os
+import pickle
+
+import numpy as np
+import torch
+from numpy.fft import fft
+from torch.utils.data import Dataset
+from torchvision import transforms
+
+import pandas as pd
+import scipy
+
+import ai8x
+
+
+class SampleMotorDataLimerick(Dataset):
+    """
+    ADXL356C
```

Comment:
We can update the docstring, can also be done after dataset repo is opened

Reply:
Updated